MANIFEST.in

```diff
@@ -1,3 +1,3 @@
 graft aiosmtpd
-include .coveragerc LICENSE *.cfg *.ini *.py *.rst *.yml
+include LICENSE *.cfg *.ini *.py *.rst *.yml
```

Comment:
`.coveragerc` has been renamed to `coverage.cfg`

Reason is explained in a59fd413